network/hostnetwork/dht_test.go

```diff
@@ -437,7 +444,7 @@ func TestBootstrapNoID(t *testing.T) {
 	<-done
 }
 
-// Create two DHTs have them connect and bootstrap, then disconnect. Repeat
+// create two DHTs have them connect and bootstrap, then disconnect. Repeat
```

Comment:
https://blog.golang.org/godoc-documenting-go-code

«The convention is simple: to document a type, variable, constant, function, or even a package, write a regular comment directly preceding its declaration»